src/main.ts

```diff
@@ -160,6 +161,25 @@ ipcMain.handle('shell.openExternal', (event, data) => {
   return shell.openExternal(data)
 })
 
+ipcMain.handle(
+  'take.screenshot',
+  async (event, data: { width: number; height: number }) => {
+    const sources = await desktopCapturer.getSources({
+      types: ['window'],
+      thumbnailSize: { width: data.width, height: data.height },
+    })
+
+    for (const source of sources) {
+      if (source.name === 'Zoo Modeling App') {
```

Comment:
Can this be a constant so that we refer to the same title that is used when creating the window?

Reply:
https://github.com/KittyCAD/modeling-app/pull/3917/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR175-R176

I've updated this to use the `productName` from `package.json` since that is what is used in `electron-builder`.
I think this patter is okay.

We do not use the API defined in [Browser-Window](https://www.electronjs.org/docs/latest/api/browser-window) to set the `title` of the browser window. So we don't have another reference within the code base. It is only defined in the `package.json`.